packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -163,37 +163,36 @@ export const DashboardReportTabView = ({
       </StickyTabBarContainer>
       <DashboardSection ref={topRef}>
         <FetchLoader isLoading={isLoading} isError={isError} error={error}>
```

Comment:
The only actual change here is chaning: `subDashboards &&
            subDashboards.map` to `subDashboards?.map`